anime_downloader/commands/watch.py

```diff
@@ -83,6 +88,15 @@ def command(anime_name, new, update_all, _list, quality, remove,
         for anime in animes:
             watcher.update_anime(anime)
 
+    if mal_import:
+        PATH = anime_name #Click seems to get the first argument as it's user input?
+        if PATH:
+            query = PATH
+        else:
+            query = click.prompt('Enter the file path for the MAL .xml file', type=str)
```

Comment:
I'd like to see a file check here, reprompt if the file is not found.

Reply:
done that, without a reprompt.